$NetBSD: patch-examples-run_All,v 1.1 2026/07/26 06:14:13 rillig Exp $

Adjust path of executable file, assuming it is in the PATH.

--- examples/Run_all.orig	Tue Oct 10 12:00:04 2000
+++ examples/Run_all
@@ -1,4 +1,4 @@
-#
+#!/bin/csh
 #
 #  Run a set of otter jobs ($1/*.in) and compare the
 #  results ($1/*.out) to $1/*.ppro180.
@@ -18,7 +18,7 @@
 if ($#argv >= 2) then
    set OTTER=$2
 else
-   set OTTER=../../source/otter
+   set OTTER=otter
 endif
 
 if ($#argv >= 3) then
